packages/react-components/src/components/utils/filter-tree/filter.tsx

```diff
@@ -19,7 +19,7 @@ export class Filter extends React.Component<FilterProps, FilterState> {
         };
         this.resizeObserver = new ResizeObserver(entries => {
             this.setState({
-                width: entries[0].contentRect.width
+                width: entries[0].contentRect.right
```

Comment:
Is this really needed? It's seems enough with just the css change below. WDYT?

Reply:
Then the filter is not as wide as it should be. The width of the filter text box should not be influenced by the left margin of the tree column.